iaac/terraform/aws-infra/ebs-csi-driver-irsa/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "iam_role_arn" {
+  description = "ARN of IAM role"
+  value       = module.ebs_csi_driver_irsa.iam_role_arn
+}
+
+output "iam_role_name" {
+  description = "Name of IAM role"
+  value       = module.ebs_csi_driver_irsa.iam_role_name
+}
+
+output "iam_role_path" {
```

Comment:
nit: remove unused outputs? ok to leave if you already tested

Reply:
got rid of it